com.unity.render-pipelines.universal/Runtime/Passes/MainLightShadowCasterPass.cs

```diff
@@ -209,9 +209,10 @@ void SetupMainLightShadowReceiverConstants(CommandBuffer cmd, VisibleLight shado
             float invHalfShadowAtlasWidth = 0.5f * invShadowAtlasWidth;
             float invHalfShadowAtlasHeight = 0.5f * invShadowAtlasHeight;
             float softShadowsProp = softShadows ? 1.0f : 0.0f;
+            float shadowDistance = (GraphicsSettings.currentRenderPipeline as UniversalRenderPipelineAsset).shadowDistance;
```

Comment:
It looks like I need the RenderData to get access to the CameraData which is given in the setup. Do you mean I should store the RenderData .cameraData.maxShadowDistance as an attribute for the render pass to use instead of the shadowDistance from the URP?

Reply:
This should have been fixed now